src/main/java/Duke.java

```diff
@@ -17,31 +17,50 @@ public static void main(String[] args) {
+                                + "Now you have " + listOfStuff.size() + " tasks in the list");
+                    }
+                } else if (ans.substring(0, 8).equals("deadline")) {
+                    if (ans.length() == 8) {
+                        DukeException ex = new DukeException(ans.substring(0, 8));
+                        System.out.println(ex.toString());
+                    } else {
+                        Deadline d = new Deadline(ans.substring(9, ans.indexOf('/') - 1),
+                                ans.substring(ans.indexOf('/') + 4));
+                        listOfStuff.add(d);
+                        System.out.println("Got it. I've added this task:" + "\n" + " " + d.toString() + "\n"
+                                + "Now you have " + listOfStuff.size() + " tasks in the list");
+                    }
+                } else if (ans.substring(0, 5).equals("event")) {
+                    if (ans.length() == 5) {
+                        DukeException ex = new DukeException(ans.substring(0, 5));
+                        System.out.println(ex.toString());
+                    } else {
+                        Event e = new Event(ans.substring(6, ans.indexOf('/') - 1),
+                                ans.substring(ans.indexOf('/') + 4));
+                        listOfStuff.add(e);
+                        System.out.println("Got it. I've added this task:" + "\n" + " " + e.toString() + "\n"
+                                + "Now you have " + listOfStuff.size() + " tasks in the list");
+                    }
                 }
-            }else if (ans.substring(0, 4).equals("done")) {
-                listOfStuff.get(Integer.parseInt(ans.substring(5)) - 1).markAsDone();
-            }else if (ans.substring(0, 4).equals("todo")){
-                Todo t = new Todo(ans.substring(5));
-                listOfStuff.add(t);
-                System.out.println("Got it. I've added this task:" + "\n" + " " + t.toString() + "\n"
-                    + "Now you have " + listOfStuff.size() + " tasks in the list");
-            }else if (ans.substring(0, 8).equals("deadline")){
-                Deadline d = new Deadline(ans.substring(9, ans.indexOf('/') - 1),
-                        ans.substring(ans.indexOf('/') + 4));
-                listOfStuff.add(d);
-                System.out.println("Got it. I've added this task:" + "\n" + " " +d.toString() + "\n"
-                        + "Now you have " + listOfStuff.size() + " tasks in the list");
-            }else if (ans.substring(0, 5).equals("event")){
-                Event e = new Event(ans.substring(6, ans.indexOf('/') - 1),
-                        ans.substring(ans.indexOf('/') + 4));
-                listOfStuff.add(e);
-                System.out.println("Got it. I've added this task:" + "\n" + " " + e.toString() + "\n"
-                        + "Now you have " + listOfStuff.size() + " tasks in the list");
+            }catch (Exception ex){
```

Comment:
Is it better to place a white space between the close bracket and the open curly brace?